README.md

```diff
@@ -259,6 +259,7 @@
 - [Microsoft Remote Desktop Connection Client](https://www.microsoft.com/en-US/download/details.aspx?id=18140) - Remote Desktop Connection Client lets you connect from your Macintosh computer to a Windows-based computer and then work with applications and files on the Windows-based computer.
 - [Site Sucker](http://ricks-apps.com/osx/sitesucker/) - Automatically download websites from the Internet.
 - [ShiftIt](https://github.com/fikovnik/ShiftIt) - Managing windows size and position. [![Open-Source Software][OSS Icon]](https://github.com/fikovnik/ShiftIt)
+- [SmartCapsLock](https://kishanbagaria.com/smartcapslock/) - Makes the <kbd>Caps Lock</kbd> key smarter, so that when that key accidentally gets activated and you START YELLING even though you don't want to, you can just select the yelling-text and press <kbd>Caps Lock</kbd> again to <s>stop yelling</s> instantly fix its case instead of typing everything all over again.
```

Comment:
@KishanBagaria  Can you trim the description a little? Also please remove markups (`<kbd>` etc).